sdk/basyx/object_store.py

```diff
@@ -6,48 +6,55 @@
 # SPDX-License-Identifier: MIT
 """
 This module implements Registries for the AAS, in order to enable resolving global
-`Identifiers`; and mapping `Identifiers` to `Identifiable` objects.
+:attr:`~basyx.aas.model.base.Identifier` and mapping
```

Comment:
About the `:attr:` vs `:class:`:
I somehow locked in on repairing these docstrings, still containing the outdated basyx-python-sdk classes. In trying to achieve proper links via intersphinx I used a tool to generate every single entry currently documentated by the old documentation. By looking through that list I found out that some entries are `:attr:` type, others `:class:`.
For exmaple Identifier only exists as `:attr:`, using it with `:class:` results in the reference not being found and therefore no intersphinx link to be placed.
I attached the file in case you want to look at it yourself. 
[all_docs.txt](https://github.com/user-attachments/files/16922044/all_docs.txt)